css/style.css

```diff
@@ -181,24 +183,53 @@ input:focus{
   display: unset;
   border: 4px solid transparent;
 }
+
 input{
   background-color: white;
 }
+
 .icon-svg{
   display: flex;
   justify-content: center;
   padding-bottom: 120px;
 }
+
 .icon-svg-brend{
   margin: 0;
   padding: 0 8px;
 }
+/* What do we offer */
 
-/* sl c svg*/
 .advantages {
   height: 28.125em;
 }
 
+.advantages-width{
+  max-width: 1440px;
+  margin: 0 auto;
+}
+
+.search-p{
+  font-size: 1.1em;
+  line-height: 1.3em;
+  margin-bottom: 20px;
+  color: #FFFFFF;
+  display: flex;
+  justify-content: flex-start;
+}
+
+.search-p-text{
+  margin: 0;
+  padding-right: 210px;
```

Comment:
You can not to align your text using gaps, you can align text in this case using label or wrapping your text with search field and align it using flex